### PR TITLE
Add zig-libp2p to interop tester

### PR DESCRIPTION
**Pull request by MarcoPolo**

### CODE REVIEW

File: multidim-interop/Makefile [hunk 1]
```diff
@@ -2,8 +2,9 @@ GO_SUBDIRS := $(wildcard impl/go/*/.)
 JS_SUBDIRS := $(wildcard impl/js/*/.)
 RUST_SUBDIRS := $(wildcard impl/rust/*/.)
 NIM_SUBDIRS := $(wildcard impl/nim/*/.)
+ZIG_SUBDIRS := $(wildcard impl/nim/*/.)
```
Comment (marten-seemann) [inline multidim-interop/Makefile:5]:
```suggestion
ZIG_SUBDIRS := $(wildcard impl/zig/*/.)
```

**Review by marten-seemann — COMMENTED** [review]
🎉

**Review by thomaseizinger — APPROVED** [review]
LGTM

**Review by marten-seemann — APPROVED** [review]

**Review by mxinden — APPROVED** [review]
:tada:

**Pull request merged**